devops/auto_node/scripts/run.py

```diff
@@ -0,0 +1,300 @@
+import argparse
+import ntpath
+import os
+import time
+import stat
+import sys
+import subprocess
+import random
+import datetime
+import json
+import re
+from multiprocessing.pool import ThreadPool
+from threading import Lock
+
+import pexpect
+import requests
+from pyhmy import (
+    Typgpy,
+    json_load
+)
+from pyhmy import cli
+
+env = os.environ
+node_start_lock = Lock()
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Run a harmony sentry node')
+    parser.add_argument("private_acc_key", help="Private wallet key to be used by validator of this node", type=str)
+    parser.add_argument("--private_bls_key", help="Private BLS key to be used by node", type=str, default=None)
+    parser.add_argument("--generate_shard", default=None,
+                        help="Shard of generated bls key, only used if private key is not provided.", type=int)
+    parser.add_argument("--network", help="Network to connect to (stress, staking, partner). Default: 'staking'.",
+                        type=str, default='staking')
+    parser.add_argument("--name", help="The `name` of the validator when creating a validator.",
+                        type=str, default=f"harmony_sentry_{random.randint(0, 1e6)}")
+    parser.add_argument("--duration", type=int, help="duration of how long the node is to run in seconds. "
+                                                     "Default is forever.", default=float('inf'))
+    parser.add_argument("--beacon_endpoint", dest="endpoint", type=str, default="http://localhost:9500/")
+    parser.add_argument("--clean", action="store_true", help="Clean directory before starting node.")
+    parser.add_argument("--confirm_all", action="store_true", help="Say yes to all interaction.")
+    parser.add_argument("--active", action="store_true", help="Always try to set active when EPOS status is inactive.")
+    return parser.parse_args()
+
+
+def setup():
+    cli.environment.update(cli.download("./bin/hmy", replace=False))
+    cli.set_binary("./bin/hmy")
+
+
+def get_current_epoch(endpoint="http://localhost:9500/"):
```

Comment:
you may use a const for the default endpint.

Reply:
yep, done in latest